share-through-htb.js

```diff
@@ -373,10 +278,10 @@ function ShareThroughHtb(configs) {
             partnerId: 'ShareThroughHtb', // PartnerName
             namespace: 'ShareThroughHtb', // Should be same as partnerName
             statsId: 'SHTH', // Unique partner identifier
-            version: '2.0.0',
+            version: STR_VERSION,
```

Comment:
We use this version number internally, are you expecting the `STR_VERSION` to match with our internal version?

Reply:
It would be ideal if this version can match with your internal version, is there a variable/constant/macro that we can put here so that it aligns with your internal version number?